system-addon/lib/SectionsManager.jsm

```diff
@@ -82,7 +82,7 @@ const BUILT_IN_SECTIONS = {
 };
 
 const SectionsManager = {
-  ACTIONS_TO_PROXY: ["SYSTEM_TICK", "NEW_TAB_LOAD"],
```

Comment:
These are no longer needed as there are existing web extension apis that cover them.